blocks/sketch/index.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace Automattic\A8c\Plugins\Blocks\Sketch;
+
+define( __NAMESPACE__ . '\DEFAULT_HEIGHT', 450);
+
+
+function a8c_sketch_render( $attributes ) {
+	$strokes = $attributes['strokes'] ?? [];
+	$align   = $attributes['align'] ?? '';
+	$height   = $attributes['height'] ?? DEFAULT_HEIGHT;
+	// The class name that affects alignment is called alignwide, alignfull, etc
+	$align = $align ? " align$align" : '';
```

Comment:
Using the "supports" don't you get all this for free?

Reply:
We only get those for free when rendering the same markup saved by the `save()` method. Given that we're not going to save any significan markup for now (c47ddd8) we're recreating the markup with classes on the PHP side